Dockerfile

```
@@ -30,13 +30,15 @@ RUN yum -y install \
     yum -y update && \
     yum -y install \
         gcc \
+        git \
         mailcap \
         postgresql10 \
         which \
         ${SCL_PYTHON_VERSION} && \
     yum clean all && rm -rf /var/cache/yum && \
     echo "source scl_source enable ${SCL_PYTHON_VERSION}" > /etc/profile.d/enable_scl_python.sh && \
     source /etc/profile && \
+    pip install wheel && \
     pip install --no-cache-dir --upgrade pip setuptools
```

This could be combined:

```suggestion
    pip install --no-cache-dir --upgrade pip setuptools wheel
```

---

Great suggestion and this change has been done.